examples/kotlin-diktat/run.sh

```diff
@@ -1,8 +1,8 @@
 #!/usr/bin/env bash
 
 KTLINT_VERSION=0.39.0
-DIKTAT_VERSION=1.0.0-rc.1
-SAVE_VERSION=0.1.0-rc.1
+DIKTAT_VERSION=1.0.0-rc.2
```

Comment:
Thoughts out loud:
I don't really like, that now we have such amount of places, where same versions are stored (`build.gradle.kts` from `buildSrc`, `Versions.kt` and 2 places in `examples`), and should keep in mind it after every release.

bash can [assign variable from another file](https://stackoverflow.com/questions/20406134/read-a-bash-variable-assignment-from-other-file), however I don't really like it.
May be, we can create something better in the future